### ISSUE TITLE
syntax highlighting in code block

### ISSUE THREAD

**Issue by jsandner** [Imatic-IT/imatic-mantis-formatting#1]
It would be nice to have syntax highlighting in code blocks like we have in github markdown files or in BBCodePlus Plugin for mantis.
I guess github does it server-side, but mantis-plugin BBCodePlus does it in Javascript (prismjs).

So if you download *prism.js* and *prism.css*  from https://prismjs.com/download.html to the *files* subdirectory and
add it to the EVENT_LAYOUT_RESOURCES hook  in *ImaticFormatting.php*
~~~php
public function layout_resources_hook() {
	return '<link rel="stylesheet" type="text/css" href="' . plugin_file('styles.css') . '&v=' . $this->version . '" />'.
		   '<link rel="stylesheet" type="text/css" href="' . plugin_file('prism.css') . '&v=' . $this->version . '" />' . 
		   '<script type="text/javascript" src="' . plugin_file( 'prism.js' ) . '"></script>'
		;
~~~
it seems to work well.

**Comment by nenadalm** [comment]
Thanks. I'll look at this tomorrow.

**Comment by nenadalm** [comment]
Released as `0.0.3`. Highlighting can be also disabled should someone want to use different configuration: https://github.com/Imatic-IT/imatic-mantis-formatting#code-highlighting
